### ISSUE TITLE
Eazfuscator.net 2023.1 How to reset 30 days?

### ISSUE THREAD

**Issue by shushu789** [Plot1337/EazTrialRemover#6]
I installed eazfuscator.net 2023.1, but I modified the system time when I test other programs today, which caused the assessment of expiration cannot be used when opening Eazfuscator.net. In fact, I did n’t use it for less than 7 days. No matter how I delete all about EAZFUSCATOR about EAZFUSCATORThe files with Gapotchenko, including the registry information, cannot restore the 30 -day evaluation and trial. Is there any other way?I originally wanted to buy genuine, but helplessly I had no international credit card and could not pay.Hope to get your reply.

Thank you.

**Comment by xAlpier** [comment]
Did you find a solution?

**Comment by shushu789** [comment]
> Did you find a solution?

No, I have purchased a legitimate license. For long-term use, I don't want to spend too much time on these unnecessary things, so I bought the genuine version.

**Comment by ldstr** [comment]
At the moment you can just delete the files generated by Eaz and redownload, but this does require a lot of work. I'm planning to work on an open-source fix for this issue soon, though. I'll keep this issue open just for that reason.

**Comment by MrFade321** [comment]
Bumping this just wondering if you ever did get a work around to the 30 day time limit.

**Comment by flechering** [comment]
> At the moment you can just delete the files generated by Eaz and redownload, but this does require a lot of work. I'm planning to work on an open-source fix for this issue soon, though. I'll keep this issue open just for that reason.

ever find a work around?
